openapi/src/parts/controlled_azure_storage_container.yaml

```diff
@@ -75,7 +101,65 @@ paths:
           $ref: '#/components/responses/ServerError'
 
 components:
+  responses:
+    ClonedControlledAzureStorageContainerResponse:
+      description: Response object for clone of an Azure storage container
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/CloneControlledAzureStorageContainerResult'
+
   schemas:
+    CloneControlledAzureStorageContainerResult:
+      description: >-
+        API result class for cloning an Azure Storage Container.
+      type: object
+      properties:
+        container:
+          $ref: '#/components/schemas/ClonedControlledAzureStorageContainer'
+        jobReport:
+          $ref: '#/components/schemas/JobReport'
+        errorReport:
+          $ref: '#/components/schemas/ErrorReport'
+
+    CloneControlledAzureStorageContainerRequest:
+      description: >-
+        Values to assign to the cloned storage container for a given source container. Leave name
+        and description null to take the source object's values.
+      type: object
+      required: [ destinationWorkspaceId, jobControl ]
+      properties:
+        destinationWorkspaceId:
+          description: ID of workspace to be cloned into
+          type: string
+          format: uuid
+        cloningInstructions:
+          $ref: "#/components/schemas/CloningInstructionsEnum"
```

Comment:
I'm confused about how the cloning instructions work-- the cloning instructions are set on a container at the time that it is created. So are these the cloning instructions for the newly created (via cloning) destination container? Or are these supposed to control what is cloned from the source container?

Reply:
Is it possible to add a description to this in addition to the $ref?

Reply:
The cloning instructions are telling WSM how to do the clone: make a reference, make a new object like the old one, make a new object and copy the contents of the old one.

Reply:
Right, but the container had cloning instructions set on it at creation time. Having gotten through the rest of the PR now, I see that this is an optional override-- it would be helpful to document that.

Reply:
See this [update](https://github.com/DataBiosphere/terra-workspace-manager/blob/1b725dff35e3e1667f36eb1e7aad95f19269907f/openapi/src/parts/controlled_azure_storage_container.yaml#L145-L147) (note I didn't update the description on the cloning instructions enum that is being referenced here as this behavior may not be universally applicable).